src/arch/xtensa/gdb/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # TODO: add gdb to final binary when it's done
```

Comment:
I realized this one should be removed, because it's going to be added with this PR